src/Aspire.Hosting/ApplicationModel/EndpointReference.cs

```diff
@@ -101,6 +101,11 @@ public EndpointReferenceExpression Property(EndpointProperty property)
     /// </summary>
     public string Url => AllocatedEndpoint.UriString;
 
+    /// <summary>
+    /// Gets the display properties for this endpoint.
+    /// </summary>
+    public EndpointDisplayProperties DisplayProperties => EndpointAnnotation.DisplayProperties;
```

Comment:
Who uses this?

Reply:
ResourceSnapshotBuilder, #239, #246, #253